src/components/scatterplot/Scatterplot.js

```diff
@@ -67,9 +69,9 @@ export default function Scatterplot(props) {
       const mappedCell = mappings[mapping];
       // The negative applied to the y-axis is because
       // graphics rendering has the y-axis positive going south.
-      return [mappedCell[0], -mappedCell[1], 0];
+      return [mappedCell[0], (FLIP_Y ? -1 : 1) * mappedCell[1], 0];
```

Comment:
Is this necessary? If it's all-caps, I wouldn't expect it to change... so why do we need the ternary?

Reply:
It is not necessary, I was just trying to use the constant variable name to make it more obvious what was happening, since we recently experienced this bug https://github.com/hubmapconsortium/vitessce/issues/674